packages/is-prop-valid/types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+// Definitions by: Junyoung Clare Jang <https://github.com/Ailrun>
+// TypeScript Version: 2.1
+
+declare const isPropValid: RegExp['test']
```

Comment:
I’d rather this be defined as `string => boolean` since it’s clearer and if for some reason it changes in the future to not use regex, it’d be more accurate. (I understand it won’t change the actual type)

Reply:
@mitchellhamilton OK, I will update one.